### DISCUSSION TITLE
Whats new in version 0.9.132 ?

### DISCUSSION THREAD

**Discussion by Fabian1410** [Budibase/budibase#2656] [Help] [answered]
What is the new update about?

**Comment by tobehn** [comment, marked as answer]
@Fabian1410 you can see a list of the changes here:
https://github.com/Budibase/budibase/compare/v0.9.132...develop

I think the budibase team describe all changes in a monthly update post you can find in the announcments at the end of each month.
https://github.com/Budibase/budibase/discussions/categories/announcements
